sass/style.css

```diff
@@ -379,8 +379,53 @@ h3.city {
   opacity: 0.4;
 }
 
-p.page-info {
-  width: 60%;
+/*********************
```

Comment:
OK with waiting for now but now that we have webpack we should switch to sass so we can use variables

Reply:
I've actually been writing scss in this file and webpack is compiling it, hahah. We should definitely change it to `.scss`, I hadn't taken the time to check if webpack would compile that without config changes